superset-frontend/src/dashboard/components/SliceHeaderControls.jsx

```diff
@@ -123,6 +125,9 @@ class SliceHeaderControls extends React.PureComponent {
   }
 
   handleMenuClick({ key, domEvent }) {
+    const menu = document.querySelector(
```

Comment:
let's move this const into the specific download as image case, that way it's also covered by the comment showing the hackiness

Reply:
We can't declare variables in `case`, because it breaks an eslint rule `no-case-declarations`

Reply:
seems like you can if you wrap the case clause in a block: https://eslint.org/docs/rules/no-case-declarations

I think this is still preferable, as the const isn't required outside of the case clause

Reply:
I didn't know about that! Done